### PR TITLE
Mediafield: fixed the js events binding in attached mode, the template and the js code, resolve #6172

### LINKED ISSUE #6172: MediaField: multiple fields in one contentitem cause DOM multiple Id error
Hi guys, 

I have a contenttype with multiple MediaField and I see, in my browser console, that it cause an error on DOM structure because every media field have some hidden input used for validation messages (I suppose) with the same value in the Id attribute.

If you want, I can do a PR to solve this issue: if doesn't exist any other activity that would fix on this problem.

The issue come from MediaFieldEditLocalization.cshtml file in the relative module.

PS. The best solution is to put MediaFieldEditLocalization.cshtml in the "Edit template" for all the MediaField in the contentitem but now I don't know how to do that

### PR DESCRIPTION
**Pull request by PiemP**
Fixes #6172 

Modified the MediaField code to avoid issues when add/edit content item with multiple MediaField. 

Modified the javascript code associated with the Mediafield, in attachment mode, to allow the dragging of the selected text and selected image in html editor.

See the commit's description for more details.

### CODE REVIEW

**Comment by PiemP** [comment]
related to #6172 .

File: src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js [hunk 1]
```diff
@@ -267,7 +267,7 @@ function initializeMediaApplication(displayMediaApplication, mediaApplicationUrl
               return;
             }
 
-            confirmDialog(_objectSpread(_objectSpread({}, $("#deleteFolder").data()), {}, {
+            confirmDialog(_objectSpread({}, $("#deleteFolder").data(), {
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js:270]: Before it was the opposite ;) as i think after some updates a full gulp built was not done again

But now if you update node and npm, do npm install and then a gulp build the generated line is

    confirmDialog(_objectSpread(_objectSpread({}, $("#deleteFolder").data()), {}, {

File: src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js [hunk 2]
```diff
@@ -309,7 +309,7 @@ function initializeMediaApplication(displayMediaApplication, mediaApplicationUrl
               return;
             }
 
-            confirmDialog(_objectSpread(_objectSpread({}, $("#deleteMedia").data()), {}, {
+            confirmDialog(_objectSpread({}, $("#deleteMedia").data(), {
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js:312]: Idem

File: src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js [hunk 3]
```diff
@@ -352,7 +352,7 @@ function initializeMediaApplication(displayMediaApplication, mediaApplicationUrl
...
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js:355]: Idem

File: src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js [hunk 4]
```diff
@@ -695,7 +695,7 @@ Vue.component('folder', {
         return;
       }
 
-      confirmDialog(_objectSpread(_objectSpread({}, $("#moveMedia").data()), {}, {
+      confirmDialog(_objectSpread({}, $("#moveMedia").data(), {
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js:698]: Idem

File: src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js (hunk 1 above)
Comment (PiemP) [reply src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js]: commit rebuild media files. Thanks @jtkech

**Comment by PiemP** [comment]
I don'have create an issue about it but I have talk about this behavior in #6172 . I give a gif about the other issue solved by this PR.

Description:
> Modified the javascript code associated with the Mediafield, in attachment mode, to allow the dragging of the selected text  and selected image in html editor.

Example:
no_move

**Comment by deanmarcussen** [comment]
@PiemP this looks good to me, sorry, can you merge dev again and rebuild the javascript, as we have merged your other pr

**Comment by PiemP** [comment]
Done. Thanks @deanmarcussen.

**Comment by PiemP** [comment]
updated the OrchardCore.Media scripts after the merge of the PR #6335

**Pull request merged**